python/setup.py

```diff
@@ -3,7 +3,7 @@
 
 setup(
     name='knesset-data',
-    version='0.0.9',
+    version='0.0.10',
```

Comment:
@noamelf - 
you should not bump the version in a pull request, you don't know when your pull request will be merged and to which version